scripts/unicorn.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=unicorn
+
+[Service]
+ExecStart=/opt/unicorn/app.js
```

Comment:
app.js isn't executable and doesn't start with the node path so probably will have to prefix with `node`
`node /opt/unicorn/app.js`